src/main/java/ch/uzh/ifi/hase/soprafs23/entity/User.java

```diff
@@ -38,8 +41,11 @@ public class User implements Serializable {
     @Column(nullable = false)
     private UserStatus status;
 
-    @Column(nullable = true)
-    private String allergies;
+  @Column(nullable = true)
```

Comment:
is the @Column necessary, or does it not conflict with the @ElementCollection? wondering especially whether the guestIds in Group need this annotation as well ^^'

Reply:
you mean bc it is displayed in greyish? I guess it is necessary and would leave it for now: 
![grafik](https://user-images.githubusercontent.com/91547040/234507983-06e0d60d-9573-4fe8-9ec5-aaeae0e3caa6.png)